cmd/run.go

```
@@ -31,20 +31,33 @@ import (
 const logoAttachmentName = "logo.png"
 
 //go:embed asset/email.default.html
-var emailBytes []byte
+var defaultEmailBytes []byte
 
 //go:embed asset/goeland@250w.png
 var logoBytes []byte
 
-func createEmailTemplate(_ config.Provider) (*template.Template, error) {
+func createEmailTemplate(config config.Provider) (*template.Template, error) {
 	minifier := minify.New()
 	minifier.Add("text/html", &mhtml.Minifier{
 		KeepConditionalComments: true,
 	})
+
+	emailBytes := defaultEmailBytes
+
+	templateFilename := config.GetString("email.template")
+	if len(templateFilename) > 0 {
+		var err error
+		emailBytes, err = ioutil.ReadFile(templateFilename)
+		if err != nil {
+			log.Fatalf("cannot read email template file: %v", err)

```

I would bubble up the err are we can already have many errors